torch/fx/experimental/symbolic_shapes.py

```diff
@@ -7326,6 +7326,20 @@ def _print_Symbol(self, sym: sympy.Symbol) -> str:
         return self.src_map[sym.name][0]
 
 
+def _is_non_negative_check(cond: sympy.Basic) -> Optional[str]:
+    """
+    Check if a condition (SymPy expression) is checking for non-negative values (>= 0).
+    Returns the variable name if it's a non-negative check (>= 0), None otherwise.
+    """
+    if isinstance(cond, sympy.Rel):
+        if cond.rel_op == ">=":
```

Comment:
nit: 
if cond.rel_op == ">=" and cond.rhs==0:
   return str(cond.lhs)
 return None